### DISCUSSION TITLE
How can i make the toggle button widget?

### DISCUSSION THREAD

**Discussion by appledino3604** [rdbende/Sun-Valley-ttk-theme#144] [Q&A]
hello, i wanted to know how to make toggle button, like in the image shown in readme file in sv_ttk.

**Comment by rdbende** [comment]
Sorry, this is not documented.

You can use a `CheckButton`, and apply the `Toggle.TButton` style to it, like this:

```python
toggle_button = ttk.Checkbutton(root, text="Toggle button", style="Toggle.TButton")
```
